starter-code/routes/celebrity.routes.js

```diff
@@ -0,0 +1,41 @@
+const express = require("express");
+const router = express.Router();
+require("dotenv").config();
+const CelebrityModel = require("../models/celebrity.model");
+
+router.get("/", (req, res) => {
+  CelebrityModel.find().then((celeb) => {
+    res.render("celebrity.hbs", { celeb });
+  });
```

Comment:
remember to handle your .catch for every promise. You want to send the user to another page or show them some messages when that happens. Also for you as a developer to know where problems might be occurring.